doc/Ptz.xml

```diff
@@ -932,6 +932,70 @@ Add GeoMove</revremark>
           </varlistentry>
         </variablelist>
       </section>
+      <section>
+        <title>MoveAndStartTracking</title>
+        <para>A device can support the MoveAndStartTracking operation. The pourpose of this action is to send an an atomic command to the device: move the camera to a wanted position and then delegate the PTZ control to the tracking algorithm. With the atomic command the latency is minimized. The move position is optional and can be set in three different modes:</para>
+		<para>By GeoLocation coordinates</para>
+		<para>By a preset token</para>
+		<para>By a PTZVector position</para>
+		<para>An existing Speed argument overrides DefaultSpeed of the corresponding PTZ configuration during movement to the requested position. If spaces are referenced within the Speed argument, they shall be speed spaces supported by the PTZ node.</para> 
```

Comment:
This is copied from AbsoluteMove, but anyway:

 _If spaces are referenced within the Speed argument, they shall be speed spaces supported by the PTZ node._

This is a client requirement. We normally write this is a device requirement. e.g.:

If the speed spaces isn't supported by the PTZ node a device shall fail this request.

Or if we don't want to specify the device behavior:
A device shall shall allow all speed spaces supported by the PTZ node.